### PR TITLE
Animations with only one frame are allowed now - TestCase

### PR DESCRIPTION
**Pull request by patriciog**
Testcase for https://github.com/cocos2d/cocos2d-x/pull/15366

### CODE REVIEW

**Comment by minggo** [comment]
Could you please fix conflict?

**Comment by patriciog** [comment]
Hi @minggo:

I sorry, but I hadn't my code this morning. I hope all is well now.

Thanks and best!

**Comment by minggo** [comment]
@patriciog This PR seems to have unneeded codes, and have conflict codes in the file.

File: tests/js-tests/project.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (minggo) [inline tests/js-tests/project.json:1]: conflicts contents

Comment (patriciog) [inline tests/js-tests/project.json:1]: I'm sorry @minggo.

In Github I see:

```
All checks have passed
1 successful check
@minggo Details Jenkins CI — Build successfully!
This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests.
```

How resolve this new conflict?

Best and thanks!

Comment (minggo) [inline tests/js-tests/project.json:1]: I think you can invoke `git status` to check which files have conflict and resolve the conflicts. I always use [sourcetree](https://www.sourcetreeapp.com/) on Mac. If you use windows, i think you have more tools to resolve it.

And there are some other files have the problem.

**Comment by minggo** [comment]
@patriciog And this PR contains many other's commits. I think it is better you create a new PR than fixing the problem.

**Comment by patriciog** [comment]
Hi @minggo:

Yeah. I'm going to create a new PR. Give me two days, because I've a lot of work now.

Thanks and I sorry!

**Comment by minggo** [comment]
@patriciog No problem, thanks.

**Pull request closed without merging**